unittests/model_tests/machine_graph_tests/test_machine_graph_model.py

```diff
@@ -239,6 +256,77 @@ def test_remember_machine_vertex(self):
         self.assertIn(mach3, app1.machine_vertices)
         self.assertIn(mach4, app2.machine_vertices)
 
+    def test_at_vertexc_methods(self):
```

Comment:
Is there an extra "c" in here that shouldn't be there?  Or should the function be called "test_at_vertex_c_methods"?

Reply:
fixed